File: NCAA/scripts/_kstate.mjs
import fs from 'fs';
const d = JSON.parse(fs.readFileSync('data/kalman_state.json','utf8'));
console.log('Top-level keys:', Object.keys(d));
for (const k of Object.keys(d)) {
  if (typeof d[k] !== 'object' || d[k] === null) {
    console.log(`  ${k}: ${d[k]}`);
  } else if (k === 'W' || k === 'W_var' || k === 'hyper') {
    console.log(`  ${k}:`, JSON.stringify(d[k]));
  } else {
    console.log(`  ${k}: [object with ${Object.keys(d[k]).length} keys]`);
  }
}

File: pyNCAA/scripts/_kstate.py
#!/usr/bin/env python3
# scripts/_kstate.py — Inspect Kalman state file
import json

d = json.loads(open("data/kalman_state.json").read())
print("Top-level keys:", list(d.keys()))
for k in d:
    if not isinstance(d[k], dict) or d[k] is None:
        print(f"  {k}: {d[k]}")
    elif k in ("W", "W_var", "hyper"):
        print(f"  {k}: {json.dumps(d[k])}")
    else:
        print(f"  {k}: [object with {len(d[k])} keys]")
